cost-analyzer/values.yaml

```diff
@@ -95,6 +95,7 @@ kubecostModel:
   warmSavingsCache: true
   etl: true
   maxQueryConcurrency: 5
+  utcOffset: "+00:00"
```

Comment:
LGTM! what do you think about a comment or doc reference to help users understand 1) the impact of changing and 2) how to set based on your geo?